Docs/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 ## Upcoming
 * Improvements to the CarlaDataProvider:
  - Added `spawn_actor` for a blueprint based actor creation similar to `World.spawn_actor`
+* Allow using with Python 3.12
```

Comment:
```suggestion
* Implement `strtobool` formerly included in `distutils`, which is removed in Python 3.12.
```

I am certain you checked it with 3.12, however I think its not yet 100% 3.12 ready, e.g. the `requirements.txt` file, and the core team should make this stement.
Therefore, I think we should adjust this statement to be more precise on this PR.

Reply:
Yes, this is better. I'm using this with 3.12, but without the `requirements.txt` file, so its content might need some update.